### DISCUSSION TITLE
New app for GOW

### DISCUSSION THREAD

**Discussion by Mannshoch** [games-on-whales/gow#202] [Q&A]
I'm looking around how I could add a custom application to gow.  I could not find a manual to do it.

Do I understand it right, that I'm not able to use a flatpak, appimage or snap for that?
Is lutris the proposed way for that?
What have I to do to e.g. get FreeCAD working?
[AppImage](https://github.com/FreeCAD/FreeCAD/releases/download/1.0.0/FreeCAD_1.0.0-conda-Linux-x86_64-py311.AppImage), [Flatpak](https://wiki.freecad.org/Flatpak), [SNAP](https://wiki.freecad.org/Ubuntu_Snap), [GitHub](https://github.com/FreeCAD/FreeCAD)

**Comment by ABeltramo** [comment]
Hi @Mannshoch :wave: 

Unfortunately the docs are a bit lacking in this regard. Wolf is able to automatically launch and control **docker containers** (this might change in future), so in order for you to be able to make custom apps you'll have to tip your toes into building your own custom Docker images.

There's a bit more info and an example in how to do that [here](https://games-on-whales.github.io/gow/adding-an-app.html). Feel free to ask here or over our Discord channels if you need any more help!

**Reply by Mannshoch** [reply]
In that case I would wish an enhancement, that allow to convert an appimage, flatpak or Snap into a docker image. As far I know about the content of these pakagetypes, i assume an Appimage would work best for that.

**Comment by Skerga** [comment]
I've build my Kodi image by replicating the structure of the other apps in this repo. The most important part i found was that the command line tool for starting gow the pre wolf way has also a build option that sets required env variables that are required to build the container. I'm also on the Discord and happy to help.

**Comment by Vlada3333** [comment]
helped
